apps/services/user-profile/src/app/v2/me-user-profile.controller.ts

```diff
@@ -138,4 +143,42 @@ export class MeUserProfileController {
       this.userProfileService.confirmNudge(user.nationalId, input.nudgeType),
     )
   }
+
+  @Get('/actor-profiles')
+  @Scopes(UserProfileScope.read)
+  @Documentation({
+    description: 'Get actor profiles for the current user.',
+    response: { status: 200, type: PaginatedActorProfileDto },
+  })
+  getDelegationPreferences(
```

Comment:
Bot methods should add audit log. The get function can use the `@Audit` decorator, while the patch method should use the `AuditService`.

Also the get function should be renamed to `getActorProfiles`

Reply:
Forgot about that 🙈